### PR TITLE
chore: update Release Please config for GA

### PR DESCRIPTION
**Pull request by k3llymariee**
Updating the release please config to (hopefully) allow for a 1.0.0 release. Please let me know if I've missed anything! I probably did!

### CODE REVIEW

File: .release-please-manifest.json [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {
-    ".": "0.13.0"
+    ".": "1.0.0"
```
Comment (k3llymariee) [inline .release-please-manifest.json:2]: is this right? we did already have a 1.0.0 at some point before so I'm not actually sure this will work

Comment (cwaldren-ld) [reply .release-please-manifest.json:2]: Reference for that: https://github.com/googleapis/release-please?tab=readme-ov-file#how-should-i-write-my-commits

Comment (k3llymariee) [reply .release-please-manifest.json:2]: ok cool wasn't sure!

File: release-please-config.json [hunk 2]
```diff
@@ -3,9 +3,8 @@
     ".": {
```
Comment (cwaldren-ld) [inline release-please-config.json:3]: You can also get rid of bootstrap-sha while you're at it

File: .release-please-manifest.json (hunk 1 above)
Comment (k3llymariee) [reply .release-please-manifest.json]: cool i'm going to merge this one in soon, so maybe just this? https://github.com/launchdarkly/ldcli/pull/293

**Review by cwaldren-ld — APPROVED** [review]

**Pull request merged**